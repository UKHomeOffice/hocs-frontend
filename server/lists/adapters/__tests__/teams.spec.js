const teamsAdapter = require('../teams');

describe('User Adapter', () => {

    it('should transform and sort team data', async () => {
        const mockData = [
            { displayName: 'Team B', type: 'TEAM_B' },
            { displayName: 'Team A', type: 'TEAM_A' },
            { displayName: 'Team C', type: 'TEAM_C' },
        ];

        const results = await teamsAdapter(mockData);

        expect(results).toBeDefined();
        expect(results).toMatchSnapshot();
    });
});