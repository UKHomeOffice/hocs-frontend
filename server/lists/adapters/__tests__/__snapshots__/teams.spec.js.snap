// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Adapter should transform and sort team data 1`] = `
[
  {
    "label": "Team A",
    "value": "TEAM_A",
  },
  {
    "label": "Team B",
    "value": "TEAM_B",
  },
  {
    "label": "Team C",
    "value": "TEAM_C",
  },
]
`;
